.gitignore

```diff
@@ -53,3 +53,10 @@ compile_commands.json
 SortedCFDatabase.def
 htmlcov
 .coverage
+
+#==============================================================================#
+# Ignore artifacts from ./utils/build-toolchain
+#==============================================================================#
+/swift-nightly-install/
+/swift-nightly-symroot/
+/swift-LOCAL-*.tar.gz
```

Comment:
I think "LOCAL" depends on the argument you pass to `build-toolchain`. Maybe it's better to ignore `*.tar.gz` directly.